app/controllers/pagerduty_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PagerdutyController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def callback
+    puser = User.find_by(github_id: 766800)
+
+    # return if pagerduty:create_user task has not been run.
+    unless puser
+      head :ok
```

Comment:
This will send an empty 200 response, but it doesn't stop the rest of the method from running - you should `return` here too.